app/views/LoginView.js

```diff
@@ -103,18 +104,25 @@ class LoginView extends React.Component {
 								onSubmitEditing={() => { this.password.focus(); }}
 								placeholder={this.props.Accounts_EmailOrUsernamePlaceholder || 'Email or username'}
 							/>
-							<TextInput
-								ref={(e) => { this.password = e; }}
-								style={styles.input_white}
-								onChangeText={password => this.setState({ password })}
-								secureTextEntry
-								autoCorrect={false}
-								returnKeyType='done'
-								autoCapitalize='none'
-								underlineColorAndroid='transparent'
-								onSubmitEditing={this.submit}
-								placeholder={this.props.Accounts_PasswordPlaceholder || 'Password'}
-							/>
+							<View style={{ flexDirection: 'row', borderColor: 'black' }}>
```

Comment:
- we have an component called TextInput, plz insert this logic ( `showPassword` ... etc ) inside of component and use it... 

- use an icon instead of text 

- use a position absolute and a wrap to put the icon inside of input